tools/build_package.sh

```diff
@@ -112,7 +122,12 @@ esac
 
 PACKAGE_VERSION=${1:-"stub-universe"}
 
+
+# Launch Publisher script if is defined
 if [ -n "$PUBLISH_SCRIPT" ]; then
     # All the scripts use the same argument format:
-    $PUBLISH_SCRIPT "${FRAMEWORK_NAME}" "${PACKAGE_VERSION}" "${UNIVERSE_DIR}" ${custom_artifacts}
+    publisher_log_file="/tmp/$(basename ${PUBLISH_SCRIPT}).log"
+    echo "Logs of publisher script in $publisher_log_file"
+    $PUBLISH_SCRIPT "${FRAMEWORK_NAME}" "${PACKAGE_VERSION}" "${UNIVERSE_DIR}" ${custom_artifacts} &> $publisher_log_file
```

Comment:
@r2dedios could we add a `tee` to both `stdout` and the `$publisher_log_file` here when verbose mode is set?

During development, we execute the build script hundreds of times a day, so having the `STUB_UNIVERSE_URL` and the build instructions emitted to the output really streamlines the development.

Reply:
I add a new commit to add that request